<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>612</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QWidget" name="widgetLegendDisplay" native="true">
      <property name="maximumSize">
       <size>
        <width>138</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <widget class="QLabel" name="labelColors">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>64</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TCP</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="gvColorTCP">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="backgroundBrush">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="interactive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelColors_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>64</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>UDP</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="gvColorUDP">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="interactive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelColors_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>64</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ARP</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="gvColorARP">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="backgroundBrush">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="interactive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelColors_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>64</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ICMP</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelColors_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>64</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>IGMP</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="gvColorICMP">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="backgroundBrush">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="interactive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QGraphicsView" name="gvColorIGMP">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="backgroundBrush">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="interactive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelColors_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>240</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>IPv6 Packet</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelColors_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>340</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>IPv6 Multicast</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGraphicsView" name="gvColorIPv6Packet">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>260</y>
         <width>51</width>
         <height>41</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="interactive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QGraphicsView" name="gvColorIPv6Multicast">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>390</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="interactive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelColors_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Unkown</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="gvColorUnkown">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="interactive">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>30000</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <widget class="QPushButton" name="buttonStartStopSniffing">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>41</width>
         <height>28</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditNetworkIPv4">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>0</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>192.168.0.0/16</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelNetworkIPv4">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>0</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>IPv4-Subnet:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelNetworkIPv6">
       <property name="geometry">
        <rect>
         <x>1140</x>
         <y>0</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>IPv6-Subnet:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditNetworkIPv6">
       <property name="geometry">
        <rect>
         <x>1260</x>
         <y>0</y>
         <width>351</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>fd00::/8</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonPause">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>41</width>
         <height>28</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxInterfaces">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>0</y>
         <width>401</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxPromiscMode">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>0</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Promiscious M.</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="buttonLegend">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>41</width>
         <height>28</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="Netvisix::NetView" name="widgetNetView" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Op&amp;tions</string>
    </property>
    <addaction name="actionRefresh_Interfacelist"/>
    <addaction name="actionReverseDNSLookup"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>&amp;Info</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionRefresh_Interfacelist">
   <property name="text">
    <string>&amp;Refresh Interface List</string>
   </property>
  </action>
  <action name="actionReverseDNSLookup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Reverse &amp;DNS Lookups</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Netvisix::NetView</class>
   <extends>QWidget</extends>
   <header>NetView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
